assets/js/base/components/radio-control/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Label from '../label';
+import './style.scss';
+
+const RadioControl = ( {
+	className,
+	selected,
+	id,
+	onChange,
+	options = [],
+} ) => {
+	const onChangeValue = ( event ) => onChange( event.target.value );
+	return (
+		options.length && (
+			<div
+				className={ classnames(
+					'wc-components-radio-control',
+					className
+				) }
+			>
+				{ options.map(
+					(
+						{
+							value,
+							label,
+							description,
+							secondaryLabel,
+							secondaryDescription,
+						},
+						index
+					) => (
+						<label
+							key={ `${ id }-${ index }` }
```

Comment:
I was hesitant to use that a well but I went with what used in `@wordpress/components/radio-control` I guess we can replace it with the radio value since it should be unique

Reply:
> I was hesitant to use that a well but I went with what used in @wordpress/components/radio-control 

Yikes, that seems like something that should be fixed in `@wordpress/components`.  Value seems a good option here and is the same as what calypso is doing:

https://github.com/Automattic/wp-calypso/blob/master/client/components/forms/form-radios-bar/index.jsx#L20-L40